agent/definitions.py

```diff
@@ -89,3 +91,9 @@ class Request:
     path: str = "/"
     headers: dict[str, str] | None = None
     data: bytes | None = None
+
+
+@dataclasses.dataclass
+class VulnRange:
+    min: version.Version
+    max: version.Version
```

Comment:
Should be nullable as you have something vulnerable from any min version for instance.

Reply:
fixed